### ISSUE TITLE
Fix collapse/uncollapse icon effect

### ISSUE THREAD

**Issue by Pabl0cks** [BuidlGuidl/extensions-hackathon#47]
It's not working to me, but I think it can be tackled after #45

**Comment by portdeveloper** [comment]
https://github.com/user-attachments/assets/4dbfcf8b-0e30-4eab-aabf-8666456eedb3

Hey do you mean it should be like the one on the left?

**Comment by Pabl0cks** [comment]
> Hey do you mean it should be like the one on the left?

Yeah! That looks awesome 🙌 Not important but it's nice to have it working :)
